pages/profile/list/views/index.jsx

```diff
@@ -65,7 +65,7 @@ function Filters() {
         formatter={filter => <Acronym>{selectivelyUppercase(filter)}</Acronym>}
         append={['pilh', 'pplh', 'admin']}
         prepend={['named']}
-        showAllLabel="All people"
+        showAll={{ position: 'before', label: 'Show all' }}
```

Comment:
Was the intention to change the label here?

Reply:
Nope!